src/ui/components/pages/assets/assetItem.styl

```diff
@@ -11,15 +11,77 @@
   position: relative;
   z-index: 1;
   margin-bottom: 8px;
-  cursor: pointer;
+  cursor: default;
   height: 64px;
 
 .assetData
   flex-grow: 1;
   text-align: left;
 
-.assetName
+.assetTitle
+  display: flex;
+  justify-content: flex-start;
   height: 24px;
-  max-width: 220px;
+  max-width: 180px;
+
+.assetName
+  padding-right: 4px;
```

Comment:
May be use the `gap` property on `.assetTitle` instead? It creates a gap between child elements.

Reply:
It looks like we don't need gap or padding here.

Reply:
Oh, I forgot that the `.assetTitle` has icons, so the ` gap` will be very useful here.